src/widgets/programs/programs.resource.tsx

```diff
@@ -56,33 +82,28 @@ export function updateProgramEnrollment(payload, abortController) {
   });
 }
 
-function mapToPatientProgram(data: any): PatientProgram {
-  return { ...data };
-}
-
-export function fetchPrograms(): Observable<any> {
-  return openmrsObservableFetch(
+export function fetchPrograms(): Observable<Program[]> {
+  return openmrsObservableFetch<FetchResponse<Program[]>>(
     `/ws/rest/v1/program?v=custom:(uuid,display,allWorkflows,concept:(uuid,display))`
-  ).pipe(map(({ data }) => data["results"]));
+  ).pipe(
+    map(({ data }: { data: FetchResponse<Program[]> }) => data["results"]),
+    map((results: Program[]) => results)
+  );
 }
 
-export function fetchLocations(): Observable<any> {
-  return openmrsObservableFetch(
+export function fetchLocations(): Observable<LocationData[]> {
+  return openmrsObservableFetch<FetchResponse<LocationData[]>>(
```

Comment:
It's just `Array<LocationData>` (no `FetchResponse` - that's implied).

Reply:
Same as above, amended.